Sources/StreamChat/Models/BlockedUser.swift

```diff
@@ -0,0 +1,29 @@
+//
+// Copyright © 2024 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+/// A type representing a blocked user. `BlockedUser` is an immutable snapshot of a blocked user entity at the given time.
+///
```

Comment:
This should be enough 👍 
```suggestion
/// A type representing a blocked user.
```